README.md

```diff
@@ -165,6 +166,22 @@ If you think that we should support the development of certain tools or projects
 
 If you have an idea for a project but lack the technical background to create a detailed outline, you can open an [issue](https://github.com/w3f/Grants-Program/issues/new) instead or add it to the [tech stack](docs/polkadot_stack.md) as a potentially interesting project. However, your idea will have better chances of being implemented if you can provide a project outline such that it can be picked up straight away by a capable team.
 
+## Maintenance Grants
+
+Maintenance Grants are yet another idea to get involved with the Polkadot community. If you are a user of an open-source library that has gone out of date, or you simply want to work on new features/fix bugs in these repos, we can support your contributions via a grant. We are happy to award rolling grants on a monthly basis, as long as the work done within each time period is performed to a quality standard deemed satisfactory by the grant evaluators.
+
+The process of applying for a Maintenance Grant is similar to what was already outlined above, but instead of defining very detailed deliverables for each milestone upfront, we will ask you to specify, where possible:
+- The repo(s) that need maintenance
+- Outline of why the specific project should continue being supported
+- Broad overview of the features/bugs that need development contributions
+- Max budget per month
```

Comment:
```suggestion
- An assurance that the current project owners are willing to review/accept your contributions (a note here: if you're fully taking over the project, it would make more sense for the current owners to transfer the repository to your organisation. If you can't get in touch with them, you may of course work on a fork)
- Max budget per month
```

Reply:
There are also abandoned projects, where a fork that is better maintained would suffice, IMO

Reply:
I did think that would be covered by the above, but feel free to reword it